tests/cacl/test_cacl_application.py

```diff
@@ -314,11 +314,13 @@ def get_cacl_tables_and_rules(duthost):
 def generate_and_append_block_ip2me_traffic_rules(duthost, iptables_rules, ip6tables_rules, asic_index):
     INTERFACE_TABLE_NAME_LIST = [
         "LOOPBACK_INTERFACE",
-        "MGMT_INTERFACE",
         "VLAN_INTERFACE",
         "PORTCHANNEL_INTERFACE",
         "INTERFACE"
     ]
+    incl_mgmt_if_branch = ['201911', '202012', '202111', '202205']
```

Comment:
hi Suvarna, afaik, the caclmgrd changes were not backported to 201911, 202012 and previous branches. Is this only required for master?

Reply:
the caclmgrd changes are not backported to 201911, 202012 or 202205 branches https://github.com/sonic-net/sonic-buildimage/blob/202205/src/sonic-host-services/scripts/caclmgrd#L246
So will need in master or in future branches based off master.
so we include mgmt_interface only for older branches.

Reply:
yes, but we've sonic-mgmt branch for 201911, 202012 which still has MGMT_INTERFACE

Reply:
Right, so we dont need this check here, should we assume that everyone will run this test case with the right sonic-mgmt branch corresponding to the image?

Reply:
Yes, because I don't think we can run master sonic-mgmt on older branches anymore as the tests have diverged